[Merging to release branch] Display presidential map data for 2020 (Deploy on 1/29)

patphongs opened the pull request:
## Summary

- Resolves #3486 
_Adds accordions for 2020 presidential map data files._

**This hotfix needs to be deployed on Wednesday 1/29. Please provide feedback ASAP.**

## Impacted areas of the application
List general components of the application that this PR will affect:

-  Browse data candidates tab: https://www.fec.gov/data/browse-data/?tab=candidates

## Screenshots

<img width="726" alt="Screen Shot 2020-01-27 at 8 59 21 PM" src="https://user-images.githubusercontent.com/12799132/73229487-ee04d900-4147-11ea-9668-53a6a12740cc.png">

## How to test
- [ ] Check out this branch
- [ ] Go to http://localhost:8000/data/browse-data/?tab=candidates and check accordions.
- [ ] Ensure links are correct according to this [doc's specs](https://docs.google.com/document/d/1KbPV_v_nxZQOhq75ogrPohM0bNmNyjiFj0fQV1Di1OI/edit?ts=5e2c41f8) 🔒 
- [ ] Ensure the 2020 list of candidates is accurate according to this [spreadsheet's specs](https://docs.google.com/spreadsheets/d/1_oOGQL7jF5bP1VYDuBh3KFRKXRCWPAvwkvb24kJ-CcM/edit#gid=894303291) 🔒

JonellaCulmer commented:
@patphongs We're missing the link below the accordions on the Candidates page: "Learn how this information is compiled >" @dorothyyeager @AmyKort Any edits to that text? The link would go to this page: https://www.fec.gov/campaign-finance-data/presidential-map-help/

@PaulClark2 Have you had a chance to look at the above page for edits to remove the directional language, such as "...at the lower right of the map." It's not a map for external users, so I don't think it should refer to a map per se. 

@PaulClark2 @AmyKort @dorothyyeager Any edits to the text inside each of the accordions? Screenshots for each are below. We can also move this language out of the accordions, generalize it and place it above the accordions.
<img width="734" alt="Screen Shot 2020-01-28 at 9 17 36 AM" src="https://user-images.githubusercontent.com/31663028/73271761-34d9e980-41af-11ea-8bfc-424604962cb8.png">
<img width="732" alt="Screen Shot 2020-01-28 at 9 17 44 AM" src="https://user-images.githubusercontent.com/31663028/73271762-34d9e980-41af-11ea-95c9-c5697978d868.png">
<img width="719" alt="Screen Shot 2020-01-28 at 9 17 51 AM" src="https://user-images.githubusercontent.com/31663028/73271763-34d9e980-41af-11ea-89a2-39d837480f77.png">
<img width="731" alt="Screen Shot 2020-01-28 at 9 17 58 AM" src="https://user-images.githubusercontent.com/31663028/73271764-34d9e980-41af-11ea-86da-9adb3a0fe82f.png">

@patphongs The top three accordions are not just 2020 data. Only the "Individual presidential map candidates" accordion. So you can remove the reference to "2020" candidates there in the summary text at the top of each accordion, except the last one.

Also, where it says "2020: All states" that should also be left-aligned. 
<img width="513" alt="Screen Shot 2020-01-27 at 12 20 01 PM" src="https://user-images.githubusercontent.com/31663028/73271999-a4e86f80-41af-11ea-8355-66c10523a50e.png">

Finally, I think there is supposed to be a larger bulk file for Contributions totals by state in the "All presidential map candidates" accordion.
<img width="814" alt="Screen Shot 2020-01-28 at 9 25 56 AM" src="https://user-images.githubusercontent.com/31663028/73272340-35bf4b00-41b0-11ea-866b-6ff038ad4299.png">

dorothyyeager commented:
@JonellaCulmer No edits from me.

patphongs commented:
@JonellaCulmer I think I have addressed all your feedback. In addition, after talking to @PaulClark2 , the ALL CONTRIBUTIONS section was the same as the all states links. So I just made sure there was an All states link above each state list and removed the all contributions section entirely. Can you please check again to confirm and approve? Thanks!
